src/runtime/module.cc

```diff
@@ -66,9 +66,19 @@ PackedFunc ModuleNode::GetFunction(const std::string& name, bool query_imports)
   PackedFunc pf = self->GetFunction(name, GetObjectPtr<Object>(this));
   if (pf != nullptr) return pf;
   if (query_imports) {
-    for (Module& m : self->imports_) {
-      pf = m->GetFunction(name, m.data_);
-      if (pf != nullptr) return pf;
```

Comment:
should'nt recursive call of m->GetFunction is enough? Perhaps we should add a GetFunction that has a query_import flag.

Reply:
I prefer current fix. As if we add `query_import` flag, we will add it into the basic core class `module`. However, for basic core class I would love to keep its current structure, that is every module `GetFunction` is his own module. The logic bug I think is here. That is when we have `query_import` flag, current logic here doesn't iterate every module we imported.

Reply:
After look at it again, i think we canjust do 

```c++
 for (Module& m : self->imports_) {
      pf = m->GetFunction(name, m.data_, query_imports);
     if (pf != nullptr) return pf;
}
...
```

Reply:
Unluckily we can not do this. Because `m` is a `Module`, which only have `GetFunction(const std::string& name, const ObjectPtr<Object>& sptr_to_self)`. This is what we said before if we want to add `query_imports`, we will modify the basic core class `Module`.

But you prompt me that I think we could do `pf =m.operator->()->GetFunction(name, query_imports);`. Get the `ModuleNode` and call the function. do you make sense to you?

Reply:
`m->` is a shorthand for` =m.operator->()` so it should work. We are not doing `m.GetFunction here`.
 GetFunction was overloaded for ModuleNode.

Reply:
I didn't realize we have one overload `operator->`